lib/generators/hyrax/work/templates/feature_spec.rb.erb

```diff
@@ -36,6 +40,30 @@ RSpec.feature 'Create a <%= class_name %>', js: false do
       # click_button "Create work"
 
       expect(page).to have_content "Add New <%= human_name %>"
+      click_link "Files" # switch tab
+      expect(page).to have_content "Add files"
+      expect(page).to have_content "Add folder"
+      within('span#addfiles') do
+        attach_file("files[]", "#{Hyrax::Engine.root}/spec/fixtures/image.jp2", visible: false)
+        attach_file("files[]", "#{Hyrax::Engine.root}/spec/fixtures/jp2_fits.xml", visible: false)
+      end
+      click_link "Descriptions" # switch tab
+      fill_in('Title', with: 'My Test Work')
+      fill_in('Creator', with: 'Doe, Jane')
+      fill_in('Keyword', with: 'testing')
+      select('In Copyright', from: 'Rights statement')
+
+      # With selenium and the chrome driver, focus remains on the
+      # select box. Click outside the box so the next line can't find
+      # its element
+      find('body').click
+      choose('<%= file_name %>_visibility_open')
```

Comment:
Are you sure you want ERB here?

Reply:
Yes.. This way the template is generated as `choose('generic_work_visibilty_open')` (just as in the `create_work_spec.rb` that is in Hyrax.